package.json

```diff
@@ -58,6 +58,23 @@
     "test": "node ./node_modules/vscode/bin/test"
   },
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "command-explorer",
+          "title": "Command Explorer",
+          "icon": "media/dep.svg"
```

Comment:
I feel like this should be the PowerShell logo

Reply:
This has been updated with the PowerShell Logo. Properly sized and everything 😀